src/test/java/com/solace/spring/cloud/stream/binder/meter/SolaceMeterAccessorTest.java

```diff
@@ -19,4 +19,14 @@ public void testRecordMessage(@Mock SolaceMessageMeterBinder messageMeterBinder)
         solaceMeterAccessor.recordMessage(bindingName, message);
         Mockito.verify(messageMeterBinder).recordMessage(bindingName, message);
     }
+
+    @Test
+    public void testRecordMessageProcessingDuration(@Mock SolaceMessageMeterBinder messageMeterBinder) {
```

Comment:
i would realy like it, it you add some IT-Test which tests, that the metric is available after you sent some messages.
Just give it a big window (like up to 500ms) for the measured duration so its not going to fail on slow infrastructure.